app/global-styles.js

```diff
@@ -22,4 +22,9 @@ injectGlobal`
     min-height: 100%;
     min-width: 100%;
   }
+
+  /* Darken Bootstrap's modal backdrop (note: styling outside html and body here is not recommended by API creators) */
+  .modal-backdrop.show {
```

Comment:
Appreciate the comment here. I reaaaaally don't like changing this file but I think it's OK for this case since it really is a global change.